### PR TITLE
Update 21-summarize.md

### PR DESCRIPTION
**Pull request by raynamharris**
Revised the final steps to instruction, replacing the submit a PR on a lesson to the one of three options.  I also moved it to the end so as to not consume anyone as two which one of three things they need to do.

### CODE REVIEW

File: _episodes/21-summarize.md [hunk 1]
```diff
@@ -8,9 +8,12 @@ questions:
...
 
-1.  submit a change to one of our lessons,
-2.  take part in a [discussion session](http://pad.software-carpentry.org/instructor-discussion) with experienced instructors, and
-3.  do a 5-minute live coding demo.
+1.  take part in a [discussion session](http://pad.software-carpentry.org/instructor-discussion) with experienced instructors, and
+2.  do a 5-minute live coding demo.
+3.  make a contribution to a lesson's content, exercises, or instructor's guide by doing **one of the following**:
+  a. submitting a change request to fix an existing issue
+  b. proof-reading a lesson and adding a new issue describing something to be improved
+  c. providing substantive feedback on an existing issue or PR
```
Comment (gvwilson) [inline _episodes/21-summarize.md]: I think it's confusing to have these out of order - we really want the live coding demo last.

Comment (raynamharris) [inline _episodes/21-summarize.md]: Good point.  Okay, maybe Discussion session first (very easy), contribute second, live coding last?

**Comment by raynamharris** [comment]
Actually, that's add a lot of layers of change.  I re-order it as just replacing point one, leaving two and three as they were

**Comment by gvwilson** [comment]
looks good to me.

**Pull request merged**